EnglishFilter/sections/content_blocker.txt

```diff
@@ -0,0 +1,1057 @@
+!
+! Section contains list of the rules for content blockers
+! It will be used by AdGuard for Android and AdGuard for iOS/Safari
+!
+! *** CAUTION: all rules are wrapped by AG directive. Don't remove it, and don't add another directives***
+!
+#if (ios || ext_safari || ext_android_cb)
```

Comment:
I'm not sure, but shouldn't it be:
`#if (adguard_app_ios || adguard_ext_safari || adguard_ext_android_cb)`
instead of:
 `#if (ios || ext_safari || ext_android_cb)`?

Reply:
sure. Fixed

Reply:
Ok :)
One more thing, I think it should be `!#if` and the same for `!#endif`

Reply:
done)